src/utils/kernel_name_shortener.py

```diff
@@ -125,18 +125,9 @@ def shorten_file(df, level):
                 "Could not resolve c++filt in expected directory: %s" % cpp_filt
             )
 
-        for fpath in glob.glob(workload_dir + "/[SQpmc]*.csv"):
-            try:
-                orig_df = pd.read_csv(
-                    fpath,
-                    on_bad_lines="skip",
-                    engine="python",
-                )
-                modified_df = shorten_file(orig_df, level)
-                modified_df.to_csv(fpath, index=False)
-            except pd.errors.EmptyDataError:
-                console_debug(
-                    "profiling", "Skipping shortening on empty csv: %s" % str(fpath)
-                )
-
-        console_log("profiling", "Kernel_Name shortening complete.")
+        try:
+            modified_df = shorten_file(df, level)
+            console_log("profiling", "Kernel_Name shortening complete.")
+            return modified_df
+        except pd.errors.EmptyDataError:
+            logging.debug("[profiling] Skipping shortening on empty csv")
```

Comment:
Instead let's use something like this with our `console_debug()` helper
```suggestion
            console_debug("profiling", "Skipping shortening on empty csv")
```